integration_tests/utils.py

```
@@ -144,42 +145,79 @@ def wait_for_block_time(cli, t):
-    wait_for_new_blocks(cli, 1)
-    res = cli.query_tally(proposal_id)
-    res = res.get("tally") or res
-    assert (
-        int(res["yes_count"]) == cli.staking_pool()
-    ), "all validators should have voted yes"
-    print("wait for proposal to be activated")
+    ev = find_log_event_attrs(rsp["events"], "submit_proposal", cb)
+    assert ev["proposal_messages"] == msg, rsp
+    return ev["proposal_id"]
+
+def approve_proposal(
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)